bin/cumulus.rb

```diff
@@ -622,9 +627,10 @@ def help_message
 
 puts "Reading configuration from '#{options[:config]}'" if options[:verbose]
 
-Cumulus::Configuration.init(options[:config], options[:profile], options[:autoscaling_force_size])
+Cumulus::Configuration.init(options[:config], options[:profile], options[:assume_role], options[:autoscaling_force_size])
 
 puts "Using aws profile '#{options[:profile]}'" if options[:verbose]
+puts "Using assume role '#{options[:assume_role]}'" if options[:verbose]
```

Comment:
Maybe, the CLI config uses "assume_role = arn" which I had thought would be pulled in when using that profile.